src/Uno.Toolkit.UI/Behaviors/ResourceExtensions.cs

```diff
@@ -36,7 +36,11 @@ private static void OnResourcesChanged(DependencyObject d, DependencyPropertyCha
 		{
 			if (d is Control control && e.NewValue is ResourceDictionary newResources)
 			{
+#if !HAS_UNO
+				control.Resources = newResources.DeepClone();
```

Comment:
@Xiaoy312 can we add some tests if possible please

Reply:
i tested in a separate project... i cant run the windows sample in toolkit somehow

Reply:
also, it is pointless since we dont do windows tests, we dont even build window sample on ci...